src/ResourceManager/DeviceProvisioningServices/Commands.DeviceProvisioningServices/Cmdlet/IotDpsCertificates/GetAzureRmIotDeviceProvisioningServiceCertificateVerificationCode.cs

```diff
@@ -0,0 +1,132 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Management.DeviceProvisioningServices
+{
+    using System;
+    using System.Management.Automation;
+    using Azure.Management.DeviceProvisioningServices;
+    using Microsoft.Azure.Commands.Management.DeviceProvisioningServices.Models;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.DeviceProvisioningServices.Models;
+
+    [Cmdlet(VerbsCommon.Get, "AzureRmIoTDeviceProvisioningServiceCertificateVerificationCode", DefaultParameterSetName = ResourceParameterSet)]
```

Comment:
@anusapan I'm still in the boat that the appropriate verb for this cmdlet should be `New` instead of `Get` since you are generating a new verification code.

From the article [Approved Verbs for Windows PowerShell Commands](https://msdn.microsoft.com/en-us/library/ms714428(v=vs.85).aspx), the comments about the `New` verb:

_For this action, do not use verbs such as Create, Generate, Build, Make, or Allocate._

Reply:
Changed to New instead of Get